backend/Controllers/raceResults.js

```diff
@@ -190,11 +40,11 @@ router.get('/race-results', (req, res) => {
     return results.calendar.races[0].adjustedResults;
   };
 
-  const setTeamsDetails = async (results) => {
-    let teamsList = await getTeamsList();
-    let raceFormats = Object.keys(results.calendar.raceFormat);
+  const setTeamsDetails = (results) => {
+    let teamsList = results.calendar.teams;
+    let raceFormat = Object.keys(results.calendar.raceFormat);
```

Comment:
```suggestion
    let raceFormats = Object.keys(results.calendar.raceFormat);
```

nazwa zmiennej która jest listą powinna być liczba mnogą